website/main.py

```diff
@@ -16,66 +20,75 @@
     "yì qiū zhī hé",
     "yí mù liǎo rán",
 ]
-_characters_selector = "#table td.char"
 
-def set_table(characters):
-    for i, cell in enumerate(document.select(_characters_selector)):
-        cell.text = characters[i]
+# initial default values (a Java convention)
+cell_selector = ""
+chars_per_cell = 0
+chars_list = []
 
-set_table(initial_cards)
+# methods relying on input arguments
 
-def write_pinyin(pinyin_list):
-    for i, cell in enumerate(document.select("#table td.pinyin")):
-        cell.text = pinyin_list[i]
+def toggle_column(col_class_name):
+    for cell in document.select(col_class_name):
+        cell.style.display = "" if cell.style.display == "none" else "none"
 
-write_pinyin(initial_pinyin)
+def get_list(string, chars_per_elem):
+    l = []
+    for i in range(int(len(string) / chars_per_elem)):
+        l.append(string[i * chars_per_elem:i * chars_per_elem + chars_per_elem])
+    return l
```

Comment:
How about change the naming and implementation to this?
```suggestion
def chop(sentence, size):
    return [sentence[i:i+size] for i in range(0, len(sentence), size)]
```
(Note: If/when you plan to accept this change as-is, you can do it by one-click from github's PR page. Alternatively, you can do it manually in your local development environment, that way you can also change the caller(s) of `get_list(...)` to `chop(...)` in the same commit.)